src/js/weather.js

```diff
@@ -0,0 +1,34 @@
+import { DailyWeather } from './daily-weather';
+
+export class Weather {
```

Comment:
let's rename class to Forecast
will more suitable, I would say

Reply:
fixed